src/Illuminate/Support/Fluent.php

```diff
@@ -46,6 +46,20 @@ public function get($key, $default = null)
     }
 
     /**
+     * Explicilty set an attribute.
+     *
+     * @param  string  $key
+     * @param  mixed   $value
+     * @return $this
+     */
+    public function set($key, $value = true)
```

Comment:
`true` is a bad default for setting a value (need to follow `Illuminate\Config\Repository`). It should be either required or `null`

Again, it should be using an extended class under the `Schema` namespace as even agreed by @taylorotwell